vm/message/src/signed_message.rs

```diff
@@ -132,7 +132,7 @@ pub mod json {
     use serde::{ser, Deserialize, Deserializer, Serialize, Serializer};
 
     /// Wrapper for serializing and deserializing a SignedMessage from JSON.
-    #[derive(Deserialize, Serialize)]
+    #[derive(Debug, Deserialize, Serialize)]
```

Comment:
Do we still need this trait?

Reply:
Nope. All of these Debug traits were for checking output before making them pretty. Completely forgot to remove